meta.go

```diff
@@ -37,37 +33,25 @@ type Meta struct {
 	Hooks       []Hook
 	Output      WriteSyncer
 	ErrorOutput WriteSyncer
-
-	lvl *atomic.Int32
+	LevelEnabler
```

Comment:
Can we keep embeds at the top, separated from the other fields with a blank line?